Sources/ReadingListKit/ListLoadingPlaceholder.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import DesignKit
+
+struct ListLoadingPlaceholder: View {
+    var body: some View {
+        VStack {
+            HStack {
+                VStack(alignment: .leading) {
+                    Text("Loading Title Text That Is Long")
+                        .font(.headline)
+                        .redacted(reason: .placeholder)
```

Comment:
For anyone else looking at this pull request: I failed to see that these Text views are marked as redacted. Thus, the text itself will be covered, ending in a view that looks like a "skeleton" view.